x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/GetDataFrameTransformsStatsAction.java

```diff
@@ -138,31 +140,53 @@ public boolean equals(Object obj) {
 
     public static class Response extends BaseTasksResponse implements ToXContentObject {
         private List<DataFrameTransformStateAndStats> transformsStateAndStats;
+        private long totalCount;
 
         public Response(List<DataFrameTransformStateAndStats> transformsStateAndStats) {
```

Comment:
Should we consider expecting the count in the constructor instead of setting it via a setter? I'm worried the constructor does not stress out the need to set the count. Looking at anomaly jobs, we expect a `QueryPage` in the corresponding class. That includes the count. Should we switch to that?